src/plugins/intel_gpu/src/graph/deconvolution.cpp

```diff
@@ -70,17 +60,14 @@ layout deconvolution_inst::calc_output_layout(deconvolution_node const& node) {
                                        0,
                                        "User-defined size of output layout must be positive (>= 1)");
 
-        tensor output_size(desc->output_size.batch[0],
+        tensor output_size(input_layout.batch(),
```

Comment:
Why not output batch?

Reply:
The value is the same, but the `output_size` parameter will be removed eventually